packages/container/README.md

```diff
@@ -0,0 +1,20 @@
+# @kaizen/container
+
+Container combines all the boilerplate and setup to ensure Kaizen components are consistent across different codebases. This includes:
+* Theme management
+* Browser normalisation 
+* Fonts
+* CSS Polyfills
+
+To use, wrap App in the React component: 
+
+```
+// App.tsx
+import KaizenContainer from "@kaizen/container"; 
```

Comment:
We seem to often use named exports rather than default exports. Did we want to standardize on this?

Reply:
Should we also document the `themeManager` export?

Reply:
Yeah, good call. The defacto standard in Kaizen is named exports, I can't recall why I made this a default export at the time.